CHANGELOG.md

```diff
@@ -43,6 +43,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ### API Breaking
 
 * (transfer) [\#1250](https://github.com/cosmos/ibc-go/pull/1250) Deprecate `GetTransferAccount` since the `transfer` module account is never used.
+* (channel) [\#1283](https://github.com/cosmos/ibc-go/pull/1283) `OnChanOpenInit` now returns a version string in line with the latest [spec changes](https://github.com/cosmos/ibc/pull/629).  
```

Comment:
I would add that it may return a default version string if relayers pass in empty version

Reply:
I left this out as I don't think the default is applicable to all modules.

Reply:
Hence the word `may`, i think it could still be useful but maybe we want to leave that suggestion elsewhere (e.g. release notes or something)